symfony/framework-bundle/4.4/config/services.yaml

```diff
@@ -19,7 +19,6 @@ services:
             - '../src/DependencyInjection/'
```

Comment:
I'd also remove `DependencyInjection` as there is no convention for it at the project level (it's the case for bundles). Furthermore, we can currently use the same Kernel class to define/handle a DI extension.

Reply:
I think it's sensible to leave `DependencyInjection` in here. It's still the place I'd put my compiler passes in and ignoring the directory here makes it clear that those passes should not be registered as services.